doc/en/components/inputs/select.md

```diff
@@ -0,0 +1,164 @@
+           github-src="inputs/select/header">
+</code-view>
+
+### Group
+
+Multiple `igc-select-item`'s can be placed between the opening and close brackets of an `igc-select-group` component so that users can visually group them together. The `igc-select-group` can be labelled via its `label` slot and disabled via its `Disable` property.
+
+> [!NOTE]
+> Keep in mind that if a select group is disabled, you cannot enable separate items of it.
+
+<code-view style="height: 380px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/inputs/select-group"
+           alt="$Platform$ Select Group Example"
+           github-src="inputs/select/group">
+</code-view>
+
+## Validation
+
+In addition, the `igc-select` supports most of the `igc-input` properties, such as `required`, `disabled`, `autofocus`, `autocomplete`, etc. The component also exposes two methods bound to its validation:
+
+- reportValidity() - checks for validity and shows a validation message if the component is invalid.
+- setCustomValidity() - sets a custom validation message. If the message is not empty, the field will be considered invalid.	
+
+## Keyboard Navigation
+
+When the select is focused and the list of options is **not visible**:
+
+- Open the `igc-select` using the `ALT + Up/Down Arrow` combination or by clicking on the `Space` or the `Enter` key.
+- Close the `igc-select` using the `ALT + Up/Down Arrow` combination or any of the `Enter`, `Space`, `Esc` or `Tab` keys.
+- Using the `Up/Left Arrow` keys will select the previous item in the list.
+- Using the `Down/Right Arrow` keys will select the next item in the list.
+- Using the `Home` or `End` keys will select the first or last item in the list.
+- Typing characters will query the list of items and select the one that most closely matches the current user input.
+
+When the select is focused and the list of options is **visible**:
+
+- Using the `Enter` or `Space` keys will select an item and close the list.
+- Using the `Up/Left Arrow` keys will activate the previous item in the list.
+- Using the `Down/Right Arrow` keys will activate the next item in the list.
+- Using the `Home` or `End` keys will activate the first or last item in the list.
+
+> [!NOTE]
+> The `igc-select` component supports only **single** selection of items.
+
+## CSS Parts
+
+Part name | Description
+---------|------------
+`container` | The main wrapper that holds all main input elements.
+`prefix`  | A prefix wrapper that renders content before the input.
+`suffix` | A suffix wrapper that renders content after the input.
+`helper-text` | A helper-text wrapper that renders content below the input.
+
+<!-- WebComponents -->
+
+## Styling
+
+The $ProductName$ Select component is built up entirely by the `igc-input` and the `igc-dropdown` components. Therefore, if you want to change the `igc-select` appearance you have to use the CSS parts exposed by the other two components.
```

Comment:
This is only true for the dropdown component as we overwrite its template in the render function of the `igc-select` component. The `igc-input`, however, exports only a few of its parts that are directly accessible on the `igc-select` component via the `::part` selector.